frontend/amundsen_application/base/base_table_quality_client.py

```diff
@@ -0,0 +1,19 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from abc import ABCMeta, abstractmethod
+
+
+class BaseTableQualityClient(metaclass=ABCMeta):
+    """
+    A Table Quality interface
+    """
+
+    @abstractmethod
+    def get_table_quality_checks(self, *, table_key: str) -> bytes:
```

Comment:
In the spirit making helpers more entity-agnostic, could we make this `get_minimal_checks` and have the argument be `entity_key`. See my other comment re "minimal"